### PR TITLE
Set Total deposit change logic

### PR DESCRIPTION
**Pull request by rorolopetegui**
Get Channel total deposit before sending the new deposit. It was trying to deposit with the actual balance of the channel, and the balance includes payments. This was making wrong operations, and getting errors.

### CODE REVIEW

File: src/actions/channelsActions.js [hunk 1]
```diff
@@ -85,6 +85,17 @@ export const depositChannel = (
     balance
```
Comment (marcosmartinez7) [inline src/actions/channelsActions.js]: Why this function receives the `balance ` parameter if it does not use it?

File: src/actions/channelsActions.js [hunk 2]
```diff
@@ -85,6 +85,17 @@ export const depositChannel = (
...
 ) => async (dispatch, getState) => {
     await getTokenApp("depositChannel");
+    let total_deposit = "";
```
Comment (marcosmartinez7) [inline src/actions/channelsActions.js]: Lets use the correct data types, this is an amount not a string.

File: src/actions/channelsActions.js [hunk 3]
```diff
@@ -85,6 +85,17 @@ export const depositChannel = (
...
+    await client
+    .get(`/api/v1/channels/${tokenAddress}/${partnerAddress}`)
+    .then(response => {
+        total_deposit = response.data.total_deposit;
+    })
+    .catch(error => {
+        console.log(JSON.stringify(error));
```
Comment (marcosmartinez7) [inline src/actions/channelsActions.js]: Remove the `console.log`

File: src/actions/channelsActions.js [hunk 4]
```diff
@@ -94,7 +105,7 @@ export const depositChannel = (
                             totalDeposit,
```
Comment (marcosmartinez7) [inline src/actions/channelsActions.js:102]: This seems to be bad, youre adding the total_deposit value to itself.

**Review by marcosmartinez7 — CHANGES_REQUESTED** [review]

File: src/actions/channelsActions.js (hunk 4 above)
Comment (rorolopetegui) [reply src/actions/channelsActions.js:102]: Changed the total_deposit for channel_deposit.

**Review by marcosmartinez7 — APPROVED** [review]
utACK

**Pull request merged**